### DISCUSSION TITLE
Is the input in Recurrent Connection and Spiking LSTM examples real value?

### DISCUSSION THREAD

**Discussion by Yang-J-LIN** [fangwei123456/SpikingFlow#241] [Q&A]
I am trying to use SNN for sequential data so I have a look at examples of recurrent connection and spiking LSTM. However, it seems that the input to the SNN is real value instead of spike trains. Is that correct? If so, how does it perform if the input is changed to latency or rate-based coding input?

**Comment by fangwei123456** [comment]
> it seems that the input to the SNN is real value instead of spike trains. Is that correct? 

The user is free to input spikes or float values. To get spikes, the user can use the encoder.

> how does it perform if the input is changed to latency or rate-based coding input?

It depends on how the user design and train the SNN.

Refer to https://github.com/fangwei123456/spikingjelly/issues/182 .
